pkg/sensorupgrader/namespace.go

```diff
@@ -0,0 +1,35 @@
+package sensorupgrader
+
+import (
+	"os"
+
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/namespaces"
+	"github.com/stackrox/rox/pkg/satoken"
+)
+
+var (
+	log = logging.LoggerForModule()
+)
+
+// GetSensorNamespace tries to read the sensor namespace out of the service account token dir, POD_NAMESPACE if that
+// fails, and defaults to Stackrox otherwise.
+func GetSensorNamespace(params ...bool) string {
```

Comment:
> the PR is currently not in a state where I want to merge it yet

I would appreciate marking it as draft. I once found this PR and felt bad that I have not reviewed it (the review was requested from Maple). When one would start looking at it, it would be clear that some things still need to be finished, thus draft fits perfect to its state.